src/lib/_fonts.scss

```diff
@@ -0,0 +1,32 @@
+@mixin printRedHatFont(
```

Comment:
Why "_fonts", and not just "fonts"?

Reply:
A sass file starting with an underscore is a partial. It is a good way to keep your styles separated into logical sections. These files all get merged on compilation when you use @import.
https://sass-lang.com/guide

When you import a _partial.scss file like `import 'partial` an inline import takes place.
In this case there was not need to do that since I am not using variables from outside this file, but I think it's a good practice.

Reply:
Agreed. Thanks for the explanation!